Cargo.lock

```diff
@@ -910,12 +895,12 @@ dependencies = [
  "anyhow",
  "clap",
  "dialoguer",
- "essential-constraint-asm 0.3.1",
- "essential-constraint-vm 0.3.0",
+ "essential-constraint-asm",
+ "essential-constraint-vm",
  "essential-hash 0.2.0",
  "essential-sign 0.2.0",
- "essential-state-asm 0.3.1",
- "essential-state-read-vm 0.3.0",
```

Comment:
Just noticed these old deps are still here. It looks like they're pulled in via `essential-debugger`, so we might need to give this an update too, I'll have a look into this now.

Reply:
Instead, I've removed the dep from `essential-app-utils` as it was unused.

We already bring the debugger CLI in via nix input and provide it to the developer shell. We should update it, but it looks like it hasn't been updated in a few months so might deserve its own PR.